python/paddle/nn/functional/activation.py

```diff
@@ -150,14 +150,16 @@ def elu_(x: Tensor, alpha: float = 1.0, name: str | None = None) -> Tensor:
 
 
 def gelu(
-    x: Tensor, approximate: bool = False, name: str | None = None
+    x: Tensor, approximate: str | bool = False, name: str | None = None
```

Comment:
```suggestion
    x: Tensor, approximate: Literal["tanh", "none"] | bool = False, name: str | None = None
```

参考 https://www.paddlepaddle.org.cn/documentation/docs/zh/dev_guides/style_guide_and_references/type_annotations_specification_cn.html#shiyonggengjiamingquedeleixingyitigonggenghaodetishixiaoguo 使用 `Literal` 吧，下同